sandcastle/api.py

```diff
@@ -185,6 +185,10 @@ def create_pod_manifest(self, command: Optional[List] = None) -> dict:
             "name": self.pod_name,
             "env": env_image_vars,
             "imagePullPolicy": "IfNotPresent",
+            # npm loves to make beautiful terminal output full of colors and sunshine
+            # which would look hideous in logs, so we should allocate a TTY
+            # unless this breaks some tool
```

Comment:
Hm, I don't really get this explanation: I see `tty: True` commented out, but I don't understand why?

Reply:
I'll explain further